feature/security/gnsi/acctz/tests/record_subscribe_non_grpc/record_subscribe_non_grpc_test.go

```diff
@@ -0,0 +1,703 @@
+	remoteAddr := addrParts[0]
+	remotePort, _ := strconv.Atoi(addrParts[1])
+
+	resolvedAddr, err := net.ResolveTCPAddr("tcp", fmt.Sprintf("%s:%d", addr, port))
+	if err != nil {
+		t.Fatalf("failed resolving ssh destination addr, error: %s", err)
+	}
+
+	addr = resolvedAddr.IP.String()
+
+	records = append(records, rpcRecord{
+		startTime:            startTime,
+		doneTime:             time.Now(),
+		cmdType:              acctz.CommandService_CMD_SERVICE_TYPE_SHELL,
+		localIp:              addr,
+		localPort:            port,
+		remoteIp:             remoteAddr,
+		remotePort:           uint32(remotePort),
+		succeeded:            true,
+		expectedStatus:       acctz.SessionInfo_SESSION_STATUS_OPERATION,
+		expectedAuthenType:   acctz.AuthnDetail_AUTHN_TYPE_UNSPECIFIED,
+		expectedAuthenStatus: acctz.AuthnDetail_AUTHN_STATUS_UNSPECIFIED,
+		expectedAuthenCause:  "",
+		expectedIdentity:     shellUsername,
+	})
+
+	return records
+}
+
+func getDutAddr(t *testing.T, dut *ondatra.DUTDevice) string {
+	var serviceDUT interface {
+		Service(string) (*tpb.Service, error)
+	}
+
+	err := binding.DUTAs(dut.RawAPIs().BindingDUT(), &serviceDUT)
+	if err != nil {
+		t.Log("DUT does not support `Service` function, will attempt to use dut name field")
+
+		return dut.Name()
+	}
+
+	dutSSHService, err := serviceDUT.Service("ssh")
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	return dutSSHService.GetOutsideIp()
+}
+
+func TestAccountzRecordSubscribeNonGRPC(t *testing.T) {
+	dut := ondatra.DUT(t, "dut")
+
+	setupUsers(t, dut)
+
+	// https://github.com/openconfig/featureprofiles/issues/2637
+	// basically, just waiting to see what the "best"/"preferred" way is to get the v4/v6 of the
+	// dut -- for now we use this hacky work around because ssh isnt exposed in introspection anyway
+	// so... we get what we can get.
+	addr := getDutAddr(t, dut)
+
+	var records []rpcRecord
+
+	// put enough time between the test starting a nd any prior events so we can easily know where
+	// our records start
+	time.Sleep(5 * time.Second)
+
+	startTime := time.Now()
+
+	// suppose ssh could be not 22 in some cases but dont think this is exposed by introspect
+	newRecords := sendCLICommand(t, addr, 22)
+	records = append(records, newRecords...)
+
+	newRecords = sendCLICommandFail(t, addr, 22)
+	records = append(records, newRecords...)
+
+	newRecords = sendShellCommand(t, dut, addr, 22)
+	records = append(records, newRecords...)
+
+	// quick sleep to ensure all the records have been processed/ready for us
+	time.Sleep(5 * time.Second)
+
+	// get gnsi record subscribe client
+	acctzClient := dut.RawAPIs().GNSI(t).Acctz()
+
+	acctzSubClient, err := acctzClient.RecordSubscribe(context.Background())
+	if err != nil {
+		t.Fatalf("failed getting accountz record subscribe client, error: %s", err)
+	}
+
+	// this will have to move up to RecordSubscribe call after this is brought into fp/ondatra stuff
+	// https://github.com/openconfig/gnsi/pull/149/files
```

Comment:
This still requires changes in the ondatra binding [code](https://github.com/openconfig/ondatra/blob/c5b17abcd56dc1cd3187c6a0324b133c3976a71b/binding/binding.go#L225) to add or point to the new `AcctzStreamClient` before it can be incorporated in the test. Can you or @bstoll please help make this change since the original acctz service is deprecated per the [proto](https://github.com/openconfig/gnsi/blob/7a39665b0f79ad7ece43d66ddf8a7da9ddf688d0/acctz/acctz.proto#L40-L77)